### PR TITLE
chore: Add transform to handle opacity as percentage

### PR DESCRIPTION
**Pull request by RayRedGoose**
## Summary

PR adds transform to handle opacity as a percentage out of 100. 

## Release Category

Infrastructure

### CODE REVIEW

File: packages/canvas-tokens/utils/transformers/index.ts [hunk 1]
```diff
@@ -28,6 +28,12 @@ export const transforms: Record<string, Transform> = {
     matcher: filter.isSysColor,
     transformer: flatRGBAColor,
   },
+  'value/opacity': {
+    type: 'value',
+    transitive: true,
+    matcher: filter.isBaseOpacity,
+    transformer: ({value}) => `${value / 100}`,
```
Comment (alanbsmith) [inline packages/canvas-tokens/utils/transformers/index.ts:35]: I always forget that JS is cool with doing math on string numbers.

**Review by alanbsmith — APPROVED** [review]
Thanks for the fix, @RayRedGoose!

File: packages/canvas-tokens/utils/filters/index.ts [hunk 2]
```diff
@@ -39,3 +39,8 @@ export const isComposite: Matcher = ({type}) => {
 export const isNotComposite: Matcher = token => {
   return !isComposite(token);
 };
+
+export const isBaseOpacity: Matcher = token => {
+  const [level, category] = token.path;
+  return level === 'base' && category === 'opacity' && parseFloat(token.value) > 1;
```
Comment (alanbsmith) [inline packages/canvas-tokens/utils/filters/index.ts:45]: `> 1` is a very smart safety guard. 👍🏼

File: packages/canvas-tokens/utils/transformers/index.ts (hunk 1 above)
Comment (RayRedGoose) [reply packages/canvas-tokens/utils/transformers/index.ts:35]: if it's only numbers, no letters lol

**Pull request merged**